app/constants/urls.js

```diff
@@ -14,7 +14,10 @@ const APP_URLS = {
     TASKS: `${SCHEME}my.${DOMAIN}/tasks`,
     IDENTITY: `${SCHEME}my.${DOMAIN}/identity`,
     MY_STATUS: `${SCHEME}my.${DOMAIN}`,
-    API_BACKEND: `${SCHEME}api.${DOMAIN}`,
+    // API_BACKEND: `${SCHEME}api.${DOMAIN}`,hostWhitelist
+
+    API_BACKEND: `http://localhost:3000`,
+
```

Comment:
Changed?

Reply:
Yes was testing, forgot to remove, thanks for pointing it out.